src/icalendar/tools.py

```diff
@@ -1,19 +1,36 @@
+"""Utility functions for icalendar."""
+
 from __future__ import annotations
+
 import random
 from datetime import date, datetime, tzinfo
 from string import ascii_letters, digits
+from warnings import warn
 
 from icalendar.parser_tools import to_unicode
 
+from .error import WillBeRemovedInVersion7
+
 
 class UIDGenerator:
-    """If you are too lazy to create real uid's."""
+    """If you are too lazy to create real uid's.
+
+    Use https://docs.python.org/3/library/uuid.html#uuid.uuid4
```

Comment:
I assume this will be released in the next minor version. Also let's make sure the reference to Python docs renders properly.

```suggestion
    """Use this only if you're too lazy to create real UUIDs.
    
    .. deprecated:: 6.3.0

        Use the Python standard library's :function:`uuid4` instead.
```